app-misc/piper/piper-0.4.ebuild

```diff
@@ -18,13 +18,20 @@ IUSE=""
 REQUIRED_USE="${PYTHON_REQUIRED_USE}"
 
 BDEPEND="
+	${PYTHON_DEPS}
 	virtual/pkgconfig
 "
 RDEPEND="
 	${PYTHON_DEPS}
-	dev-libs/libratbag
-	$(python_gen_cond_dep 'dev-python/python-evdev[${PYTHON_MULTI_USEDEP}]')
+	>=dev-libs/libratbag-0.13
+	dev-python/lxml
+	dev-python/pycairo
```

Comment:
Modules from `dev-python/lxml` and `dev-python/pycairo` are imported, so for both of them, `$(python_gen_cond_dep ...` is needed.